src/status_im2/contexts/communities/menus/channel_options/view.cljs

```diff
@@ -0,0 +1,121 @@
+
+(defn mark-as-read
+  [id]
+  {:icon                :i/up-to-date
+   :accessibility-label :mark-as-read
+   :label               (i18n/label :t/mark-as-read)
+   :on-press            #()})
+
+(defn mute-channel
+  [id muted?]
+  {:icon                (if muted? :i/muted :i/activity-center)
+   :accessibility-label (if muted? :unmute-channel :mute-channel)
+   :label               (i18n/label (if muted? :t/unmute-channel :t/mute-channel))
+   #_:sub-label
+   #_(when muted? (str "muted for 15 minutes"))
+   :on-press            (if muted?
+                          #(unmute-channel-action id)
+                          #(mute-channel-action id))
+   :right-icon          :i/chevron-right
+   :chevron?            true})
+
+(defn fetch-messages
+  []
+  {:icon                :i/save
+   :label               (i18n/label :t/fetch-messages)
+   :on-press            #(js/alert "TODO: to be implemented, requires design input")
+   :danger?             false
+   :accessibility-label :fetch-messages
+   :sub-label           nil
+   :right-icon          :i/chevron-right
+   :chevron?            true})
+
+(defn invite-contacts
+  [id]
+  {:icon                :i/add-user
+   :accessibility-label :invite-people-from-contacts
+   :label               (i18n/label :t/invite-people-from-contacts)
+   :on-press            #(hide-sheet-and-dispatch [:communities/invite-people-pressed id])})
+
+(defn show-qr
+  [id]
+  {:icon                :i/qr-code
+   :accessibility-label :show-qr
+   :on-press            #(js/alert (str "implement action" id))
+   :label               (i18n/label :t/show-qr)})
+
+(defn share-channel
+  [id]
+  {:icon                :i/share
+   :accessibility-label :share-channel
+   :label               (i18n/label :t/share-channel)
+   :on-press            #()})
+
+(defn notifications
+  []
+  {:icon                :i/notifications
+   :label               (i18n/label :t/notifications)
+   :on-press            #(js/alert "TODO: to be implemented, requires design input")
+   :danger?             false
+   :sub-label           "All messages" ; TODO: placeholder
+   :accessibility-label :manage-notifications
+   :right-icon          :i/chevron-right
+   :chevron?            true
+   :add-divider?        false})
+
+(defn channel-opts
+  [community-id id token-gated? muted?]
+  [[(view-members id)
+    (when token-gated? (view-token-gating id))
+    (mark-as-read id)
+    (mute-channel (str community-id id) muted?)
+    (notifications)
+    (fetch-messages)
+    (invite-contacts id)
+    (show-qr id)
+    (share-channel id)]])
+
+(defn get-context-drawers
+  [{:keys [id community-id]}]
+  (let [{:keys [token-gated?]} (rf/sub [:communities/community id])
+        {:keys [muted]}        (rf/sub [:chat-by-id (str community-id id)])]
+    (channel-opts community-id id token-gated? muted)))
+
+(defn channel-options-bottom-sheet
+  [community-id id]
+  [quo/action-drawer
+   (get-context-drawers {:id           id
```

Comment:
`get-context-drawer-options`

Reply:
actually where is no need in this function at all, just move `get-context-drawers` here

Reply:
Okay